.github/workflows/create-release-pr-to-develop.yml

```diff
@@ -0,0 +1,68 @@
+name: create-release-pr-to-develop
+on:
+  push:
+    branches:
+      - 'release/*'
+
+jobs:
+  rebase-release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.MERGE_TOKEN }}
+          fetch-depth: 0
+      - name: enable merge commits
+        uses: octokit/request-action@v2.x
+        with:
+          route: PATCH /repos/{owner}/{repo}
+          owner: aws-solutions
+          repo: solution-spark-on-aws
+          allow_merge_commit: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.MERGE_TOKEN }}
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v4
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+      - name: rebase-with-release
+        run: |
+          git checkout rebase-$${{ github.ref_name }}
+          git merge origin/$${{ github.ref_name }} --no-ff --no-edit -X theirs
+          git push origin rebase-$${{ github.ref_name }}
+      - name: create-pull-request
+        uses: repo-sync/pull-request@v2
+        id: pr
+        with:
+          source_branch: "rebase-$${{ github.ref_name }}"
+          destination_branch: "develop"
+          pr_title: 'chore: merge $${{ github.ref_name }} to develop'
+          pr_template: ".github/PULL_REQUEST_TEMPLATE.md"
+          pr_label: "auto-release-pr-to-develop"
+          pr_allow_empty: false
+          github_token: ${{ secrets.MERGE_TOKEN }}
+      - name: Enable Pull Request Automerge
```

Comment:
if we are using `-X theirs` to resolve conflicts then I wonder if just creating the PR and sending out a notification is a safer option compared to automatically merge it ?

Reply:
Discussed offline !